website/docs/r/azurerm_server_vulnerability_assessment.html.markdown

```diff
@@ -0,0 +1,111 @@
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefix       = "192.168.1.0/24"
+}
+
+resource "azurerm_network_interface" "example" {
+  name                = "vm-example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  ip_configuration {
+    name                          = "vm-example"
+    subnet_id                     = azurerm_subnet.example.id
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_linux_virtual_machine" "example" {
+  name                = "vm-example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  size                            = "Standard_B1s"
+  admin_username                  = "testadmin"
+  admin_password                  = "Password1234!"
+  disable_password_authentication = false
+
+  source_image_reference {
+    publisher = "OpenLogic"
+    offer     = "CentOS"
+    sku       = "7.5"
+    version   = "latest"
+  }
+
+  os_disk {
+    caching              = "ReadWrite"
+    storage_account_type = "Standard_LRS"
+  }
+
+  network_interface_ids = [azurerm_network_interface.example.id]
+}
+
+resource "azurerm_server_vulnerability_assessment" "example" {
+  virtual_machine_id = azurerm_linux_virtual_machine.example.id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `virtual_machine_id` - (Optional) The ID of the virtual machine to be monitored by vulnerability assessment. Changing this forces a new resource to be created.
+  
+-> **NOTE:** One of either `virtual_machine_id` or `hybrid_machine_id` must be set.
+
+* `hybrid_machine_id` - (Optional) The ID of the Azure ARC server to be monitored by vulnerability assessment. Changing this forces a new resource to be created.
+  
+-> **NOTE:** One of either `virtual_machine_id` or `hybrid_machine_id` must be set.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The ID of the Vulnerability Assessment resource.
+
+## Timeouts
+
+The `timeouts` block allows you to specify [timeouts](https://www.terraform.io/docs/configuration/resources.html#timeouts) for certain actions:
+
+* `create` - (Defaults to 30 minutes) Used when creating the Advanced Threat Protection.
+* `read` - (Defaults to 5 minutes) Used when retrieving the Advanced Threat Protection.
+* `delete` - (Defaults to 30 minutes) Used when deleting the Advanced Threat Protection.
+
+## Import
+
+Server Vulnerability Assessment can be imported using the `resource id`, e.g.
+
+```shell
+terraform import azurerm_server_vulnerability_assessment.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/resource-group-name/providers/Microsoft.Compute/virtualMachines/vm-name/providers/Microsoft.Security/serverVulnerabilityAssessments/Default
+```
```

Comment:
Suggest to also add an example of Microsoft.HybridMachine.

Reply:
I do not think it is necessary since it is just to copy paste whatever ID your resource has.

Reply:
I saw an example that it lists all possible samples. It's fine to keep it for now.

Reply:
That example imports on something else then the ID full id, so I think that is different since this only supports importing on the ID.

Reply:
I'd like us to show both please